doc/nrf/ug_nrf5340.rst

```diff
@@ -567,6 +567,27 @@ See the following instructions.
    :start-after: fota_upgrades_start
    :end-before: fota_upgrades_end
 
+Simultaneous multi-image DFU
+****************************
+
+The simultaneous update of multiple images is available for testing since |NCS| v1.7.0.
+It allows the updating of both the application core and the network core in one go.
+
+To enable the simultaneous update of multiple images, set the following options:
+
+* :kconfig:`CONFIG_BOOT_UPGRADE_ONLY` - The simultaneous update of multiple images does not support network core image reversion, so you need to disable application image reversion.
+* :kconfig:`CONFIG_PCD_APP` - Enable commands exchange with the network core.
+* :kconfig:`CONFIG_UPDATEABLE_IMAGE_NUMBER` - Enable support for multiple update partitions by setting this option to ``2``.
+
+Additionally, the memory partitions for secondary images for both the application and the network core must be defined.
+For an example of such configuration, refer to :ref:`ug_thingy53`.
```

Comment:
Should this point to the application guide section?

Reply:
I also find this information confusing, because the planned additions to release notes state:
```
The support for the simultaneous multi-image DFU for nRF53 Series was added.
Refer to :ref:`ug_nrf5340` and :ref:`ug_thingy53` for more information and a configuration example.
```
But where exactly this "more information" and "configuration example" is to be found in ug_thingy53?

Reply:
Thought about it but there are two at least several places we could point to. One is information about images and update. Second is application configuration requirements.
This is why I though it would be better to create section in this rst instead of pointing directly to thiny53 ug (as I originally thought).
I think we need to review this rst files after the release and check how things fit together. Some things do not have documentation so I have nothing to refer to.

Reply:
After the release, we could probably create a new document that covers all the matters that are the same between the DK and Thingy:53.

Reply:
But is this "configuration example" in the ug_thingy53 or it is not there? You say it is, so can we point to the section where it is described there?

Reply:
Discussed off-line. I have reworded the reference. Also added few more information that should help readers to find the right door.